test/solveEquation/solveEquation.test.js

```diff
@@ -110,6 +109,7 @@ describe('solveEquation for =', function () {
     // TODO: figure out what to do about errors from rounding midway through
     // this gives us 6.3995 when it should actually be 6.4 :(
     // ['x - 3.4= ( x - 2.5)/( 1.3)', 'x = 6.4']
+    // ['( u )/( 0.3) = 4u + 6.28', 'u = -9.42'], //Commented out for bug #162
```

Comment:
I don't think you need to add the comment, it's now just another case of the bug and people in the future don't need to know it once passed

I find it so funny how it was passing because we were rounding wrong and now is failing haha